### DISCUSSION TITLE
Running Bot on win10 wsl2

### DISCUSSION THREAD

**Discussion by yurass13** [aiogram/aiogram#1537] [Q&A]
Polling is works, but `send_*` methods got an `aiogram.exceptions.TelegramNetworkError: HTTP Client says - Request timeout error` in `make_request`.

Looks like there is a proxy problem between wsl and win, but at the same time, I am a little confused by the fact that polling works fine.

System: Win10
WSL: 2.1.5.0
Python: 3.12.3
aiogram: 3.9.0
aiohttp: 3.9.5

Code sample:
```py
import asyncio
import logging
from aiogram import Bot, Dispatcher, types
from aiogram.filters.command import Command

BOT_TOKEN = '12345:Token'

logging.basicConfig(level=logging.INFO)
bot = Bot(token=BOT_TOKEN)
dp = Dispatcher()

@dp.message(Command("start"))
async def cmd_start(message: types.Message) -> None:
    await message.answer("Done!")

async def main() -> None:
    await dp.start_polling(bot)

if __name__ == "__main__":
    asyncio.run(main())
```
This doesn't seem like an aiogram issue, but it still seems pretty annoying.
Maybe someone else has encountered something similar?

**Comment by andrew000** [comment]
Can't reproduce on:
WSL version: 2.2.4.0
Python: 3.12.4
aiogram: 3.9.0
aiohttp: 3.9.5
![image](https://github.com/user-attachments/assets/675b8913-8234-4bd4-b97b-435a8ebbaa84)
![image](https://github.com/user-attachments/assets/c6f2208e-96c8-460d-bb78-088aea591cd0)
